Snippets/Bridge/Bridge_1/Configuration.cs

```diff
@@ -84,13 +84,19 @@ public void RegisterPublishers()
             asb.HasEndpoint("Shipping");
 
             var invoicing = new BridgeEndpoint("Finance.Invoicing");
-            invoicing.RegisterPublisher("Messages.OrderPlaced", "Sales");
             invoicing.RegisterPublisher(typeof(OrderBilled), "Finance.Billing");
             invoicing.RegisterPublisher<OrderShipped>("Shipping");
+            invoicing.RegisterPublisher("Messages.OrderPlaced", "Sales");
 
             asb.HasEndpoint(invoicing);
 
             #endregion
+
+            #region register-publisher-legacy
+
+            invoicing.RegisterPublisher("CreditApproved, CreditScoring.Messages, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null", "Sales");
```

Comment:
Is `, Culture=neutral, PublicKeyToken=null` needed?

Reply:
Nope, funny that you mention it as yesterday I did a commit on the bridge that removed these if they match the default values :-)
```suggestion
            invoicing.RegisterPublisher("CreditApproved, CreditScoring.Messages, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null", "Sales");
            // or, drop Culture and PublicKeyToken which are ignored by the message driven pub/sub feature
            invoicing.RegisterPublisher("CreditApproved, CreditScoring.Messages, Version=1.0.0.0", "Sales");
            
```

Reply:
@andreasohlund how about this?